backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MigrateVmCommand.java

```diff
@@ -1100,4 +1113,9 @@ public void reportCompleted() {
             super.reportCompleted();
         }
     }
+
+    @Override
+    protected VdsManager getVdsManager() {
```

Comment:
This is quite confusing, please rename to getDestinationVdsManager and use getVdsManager() for the source vds. You can then pass the manager instance to setDedicatedCpus method.